frontend/public/components/utils/_status-box.scss

```diff
@@ -29,3 +29,12 @@
   margin: 0;
   position: static;
 }
+
+.co-m-timeout-error {
+  margin-bottom: $grid-gutter-width;
+  text-align: center;
+  .co-cluster-updates__component,
+  .co-m-vert-nav & {
+    margin-top: $grid-gutter-width;
+  }
```

Comment:
So far, I have been unable to find a solution that worked in all contexts given the significant markup differences among all the places &lt;StatusBox&gt; is used and whether or not it wraps children.  In other words, it's hard to reason about the markup.  I realize the CSS isn't ideal, but neither is the markup.

Reply:
@rhamilto 

> I realize the CSS isn't ideal, but neither is the markup.

Any idea how hard it would be to fix the HTML instead of pushing the problem down to CSS?  `co-m-vert-nav` is used in exactly one place, `co-cluster-updates__component` is in a few places (although maybe it shouldn't be).

Reply:
I'd also prefer fixing the markup to having these exceptions in the styling. Those ampersands are easy to miss, and they reverse the normal logic of style rules.